components/SHeader.vue

```diff
@@ -128,11 +128,6 @@ header {
                     background-color: var(--color-green-600);
                 }
             }
-
-            &__link {
```

Comment:
Why was this removed? Is it not needed anymore?

Reply:
It was a class that wasn´t used anywhere